internal/epoch/epoch_advance.go

```diff
@@ -36,6 +37,10 @@ func shouldAdvance(bms []blob.Metadata, minEpochDuration time.Duration, countThr
 		totalSize += bm.Length
 	}
 
+	if totalSize >= totalSizeBytesThresholdHard {
+		return true
+	}
+
```

Comment:
@Lyndon-Li Please remove this check from this PR. At the moment, we are not going to change the conditions for compaction.